ios/TPSTwitterModule/TPSTwitterModule.m

```diff
@@ -0,0 +1,222 @@
+                    
+                    for (ACAccount *twAccount in accounts) {
+                        
+                        UIAlertAction* action = [UIAlertAction
+                                                 actionWithTitle:[NSString stringWithFormat:@"@%@", twAccount.username]
+                                                 style:UIAlertActionStyleDefault
+                                                 handler:^(UIAlertAction * action) {
+                                                     __strong typeof(self) strongSelf = weakSelf;
+                                                     [strongSelf createSessionFromAccount:twAccount withHandler:^(TWTRSession *session, NSError *error) {
+                                                         if (error) {
+                                                             reject(nil, nil, error);
+                                                         } else {
+                                                             NSDictionary *body = @{@"authToken": session.authToken,
+                                                                                    @"authTokenSecret": session.authTokenSecret,
+                                                                                    @"userID":session.userID,
+                                                                                    @"userName":session.userName};
+                                                             resolve(body);
+                                                         }
+                                                     }];
+                                                 }];
+                        [alert addAction:action];
+                    }
+                    
+                    UIAlertAction* webLoginAction = [UIAlertAction
+                                                     actionWithTitle:@"Log in as another user"
+                                                     style:UIAlertActionStyleDefault
+                                                     handler:^(UIAlertAction * action) {
+                                                         __strong typeof(self) strongSelf = weakSelf;
+                                                         [strongSelf webBasedLogin:resolve rejecter:reject];
+                                                     }];
+                    [alert addAction:webLoginAction];
+                    
+                    UIAlertAction* cancelAction = [UIAlertAction
+                                                   actionWithTitle:@"Cancel"
+                                                   style:UIAlertActionStyleCancel
+                                                   handler:nil];
+                    [alert addAction:cancelAction];
+                    dispatch_async(dispatch_get_main_queue(), ^{
+                        [RCTPresentedViewController() presentViewController:alert animated:YES completion:nil];
```

Comment:
I think all work with UI should be do only on main thread, I mean creating UIAlertController...